control/plugin/rpc/plugin.go

```diff
@@ -90,7 +90,13 @@ func ToConfigPolicy(reply *GetConfigPolicyReply) *cpolicy.ConfigPolicy {
 			nodes[k] = cpolicy.NewPolicyNode()
 		}
 		for key, val := range v.Rules {
-			br, err := cpolicy.NewBoolRule(key, val.Required, val.Default)
+			var br *cpolicy.BoolRule
+			var err error
+			if val.HasDefault {
+				br, err = cpolicy.NewBoolRule(key, val.Required, val.Default)
+			} else {
+				br, err = cpolicy.NewBoolRule(key, val.Required)
+			}
 			if err != nil {
 				// The only error that can be thrown is empty key error, ignore something with empty key
```

Comment:
I added warning statement [here](https://github.com/intelsdi-x/snap/pull/1150/files#diff-8dbe807662f325362a28f3160484eb3bR109). I don't have a strong opinion on it.